coalib/processes/Processing.py

```diff
@@ -751,7 +755,12 @@ def execute_section(section,
                              results (bear names are key) as well as the
                              file dictionary.
     """
-    if debug:
+    debug_bears = (False
+                   if 'debug_bears' not in section or (
+                       section['debug_bears'].value == 'False') else (
+                            typed_list(str)(section['debug_bears'])))
```

Comment:
Just thinking, what if you now specify `debug-bears = True` in a coafile. Won't you now get a typed list with `['True']` as value?

Reply:
No it will split it up like `['T', 'r', 'u', 'e']`